dashboard/app/views/peer_review_mailer/review_completed_receipt.html.haml

```diff
@@ -0,0 +1,9 @@
+%p
+  Hi
```

Comment:
Just my opinion but I'm more of a Hello person than a hi person. Also comma after name is standard I believe.